automatic/sysinternals/tools/chocolateyInstall.ps1

```diff
@@ -21,6 +21,23 @@ if (Is-NanoServer) {
   $packageArgs.checksum = '7b6fab7f0ada0bc2b0748f3bc29e93846869f9dd175a0ca070bbebc07ee5de09'
  }
 
+if (-Not $pp.NoStartMenu) {
+  Write-Verbose "Creating start menu entry"
+  set-alias 7z $Env:chocolateyInstall\tools\7z.exe
+  $linkfiles = get-childitem "$installDir" -include "*.exe" -recurse
+  $startmenufolder = "$($env:ProgramData)\Microsoft\Windows\Start Menu\Programs\Sysinternals Suite"
```

Comment:
Don't resolve the path this way, rather use
```suggestion
  $startmenufolder = Join-Path ([Environment]::GetFolderPath('CommonPrograms')) 'SysInternals Suite'
```

Valid values for `GetFolderPath` can be seen here: https://docs.microsoft.com/en-us/dotnet/api/system.environment.specialfolder?view=netframework-4.0 (only those available in .NET 4.0 can be used)

Reply:
I've seen all over the place, but you are right of course.  
Adding an example to https://chocolatey.org/docs/helpers-install-chocolatey-shortcut  and adding a check to the validation could improve the situation. I'd still prefer https://github.com/chocolatey/choco/issues/395 to be fixed, though. :)